lib/open_food_network/reports/helper.rb

```diff
@@ -0,0 +1,42 @@
+module OpenFoodNetwork
+  module Reports
+    module Helper
```

Comment:
I see you tried to move everything into an helper. I dont think this is good practice, these is OFN's core business logic and needs to be kept consistent in the core, not in a report helper.
I suggest moving this to the existing models and if not possible, move both the existing ones in the models and these methods into services.